src/Skia/Avalonia.Skia/DrawingContextImpl.cs

```diff
@@ -676,6 +705,37 @@ internal PaintWrapper CreatePaint(SKPaint paint, IBrush brush, Size targetSize,
                 return paintWrapper;
             }
 
+            if (brush is IExperimentalAcrylicBrush acrylicBrush)
+            {
+                var tintOpacity =
+                    acrylicBrush.BackgroundSource == AcrylicBackgroundSource.Digger ?
+                    acrylicBrush.TintOpacity : 1;
+
+                var noiseOpcity = 0.06 * brush.Opacity;
+
+                var tintColor = acrylicBrush.TintColor;
+                var excl = new SKColor(255, 255, 255, (byte)(255 * 0.1));
+                var tint = new SKColor(tintColor.R, tintColor.G, tintColor.B, (byte)(255 * ((tintColor.A / 255.0) * acrylicBrush.Opacity)));
+
+                tint = SimpleColorBurn(excl, tint);
+
+                using (var tintShader = SKShader.CreateColor(tint))
+                using (var noiseShader =
+                    SKShader.CreatePerlinNoiseTurbulence(15.876f, 15.876f, 2, 0.76829314f)
+                    .WithColorFilter(CreateAlphaColorFilter(noiseOpcity)))
+                using (var compose = SKShader.CreateCompose(tintShader, noiseShader))
```

Comment:
@danwalmsley I have tried with noise texture and noise looks good now. I don't really think, if it's great idea to ship png file with every application as WinUI does (or ship embedded instead?), but I am not sure, if it is easy to generate consistently good noise in runtime.
I have compared this brush with "Acrylic" application from MS Store and WinUI's Xaml Controls Gallery. Also I have set AccentPolicy.GradientColor value to 0x01000000 for windows (in other way acrylic background with transparent brush looks too "white"). But I haven't tested it with luminosity, because Acrylic app and WinUI XAML app doesn't have options for this. I can create simple one tomorrow evening.

With better noise and app background it already looks great for me (except gradient on top from undocumented api). But if we want acrylic with same effect as in UWP (I don't think if this is necessary), I have found also two small issues:
1. With UWP Acrylic I can set any TintOpacity, and with 100% value it will looks like solid color without any effects. But with Avalonia TintOpacity=100% still is partially transparent (it seems like GetTintOpacityModifier should work only in pair with GetEffectiveLuminosityColor, and returns 1.0d on older OS, where luminosity is not available).
2. In UWP Brush with TintOpacity=100% doesn't have 10% of white - it still has clear solid color (FF0000 for Red tint). So it seems like "excl" (10% of white) should also depend on TintOpacity, or also needs luminosity?

```suggestion
                var noiseOpcity = 0.02 * brush.Opacity;

                var tintColor = acrylicBrush.TintColor;
                var excl = new SKColor(255, 255, 255, (byte)(255 * 0.1));
                var tint = new SKColor(tintColor.R, tintColor.G, tintColor.B, (byte)(255 * ((tintColor.A / 255.0) * acrylicBrush.Opacity)));

                tint = SimpleColorBurn(excl, tint);

                using (var tintShader = SKShader.CreateColor(tint))
                using (var bitmap = SKBitmap.Decode("noise.png"))
                using (var noiseShader = SKShader.CreateBitmap(bitmap, SKShaderTileMode.Repeat, SKShaderTileMode.Repeat)
                    .WithColorFilter(CreateAlphaColorFilter(noiseOpcity)))
                using (var compose = SKShader.CreateCompose(tintShader, noiseShader))
```

Reply:
Different TintOpacity values, but almost same acrylic. And in Avalonia 10% of white is more noticeable.
![image](https://user-images.githubusercontent.com/3163374/83346198-604d1c00-a2e8-11ea-80b1-cc9afdaaee2b.png)

Reply:
Changed the way acrylic works significantly... we will not be able to achieve identical results to UWP unfortunately. (At least for BackDrop acrylic, inapp acrylic will be possible in future and identical results to uwp is reasonable)

We can also make the result fairly consistent across platforms and blur radius (which is platform dependent) will need some adjustment factor depending on what the platform provides in the window.

So we have:

```
<HostBackDrop> - not accessible
<HostProvidedBlur> - not controlable
<TracingPaperLayer> - #xx7f7f7f makes the backdrop more opaque on platforms that cant do this for us.
<TintLayer> - Adds the tint colour, blended with the tracing paper layer.
<NoiseLayer> - Adds the texture blended with the TintLayer.
```